### DISCUSSION TITLE
Delete files in folder A that exists in folder A and folder B

### DISCUSSION THREAD

**Discussion by freeze-out** [WinMerge/winmerge#2571] [Q&A] [answered]
Is there any way to compare files that exist in A and B without comparing the folder tree structure and the file names so I can delete the files in A?

```
- A
-- 1.txt (Content: ABC)

- B
-- Any
--- 2.txt (Content: ABC)
```

`A/1.txt` can be deleted.

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, WinMerge does not have the ability to match files that have different folder or file names but the same content.
However, you can use the current functionality to find files that have matching content.

1. Disable Tree Mode in the Folder Compare window.
![image](https://github.com/user-attachments/assets/83ade1ef-7aaa-40f3-be51-b0b3281f6d36)

2. Add an MD5(Move) column.

![image](https://github.com/user-attachments/assets/24eb7b83-3406-438f-b5e0-76607838a8d9)
![image](https://github.com/user-attachments/assets/bea64400-22a5-47c0-800b-3d7e728e6304)
![image](https://github.com/user-attachments/assets/b7100a7f-9ce1-4e60-a857-505443c8b167)
![image](https://github.com/user-attachments/assets/d80be175-bb03-4705-a1c5-630c59f21634)

3. Press F5 to recompare. The MD5(Move) column should show something like `Group**`. If these values ​​are the same, the files match. Sorting by this column will make it easier to find them.

![image](https://github.com/user-attachments/assets/b17bc06f-fa65-4892-830e-40aa7e1a7f12)
